### PR TITLE
Applications refactor

### PR DESCRIPTION
**Pull request by fieldstyler**
Complete User Story 36 for applications.

Refactored all applications file for less lines of code

### CODE REVIEW

File: app/views/pets/show.html.erb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
   <% if @pet.adoption_status == "Pending" %>
-  <h3>On hold for <%= @app.name %></h3>
+  <h3>On hold for <%= link_to @app.name, "/applications/#{@app.id}" %></h3>
```
Comment (sarynm12) [inline app/views/pets/show.html.erb:3]: Love it.

File: spec/features/applications/show_spec.rb [hunk 2]
```diff
@@ -55,14 +55,6 @@
...
 
-  it 'displays message when no apps are present' do
-
-    visit "/pets/#{@pet2.id}"
-
-    # Not sure how to test this with a dropdown menu
-    # expect(page).to have_content("There are no current applications for this pet.")
-  end
-
   it 'can approve application for more than one pet' do
```
Comment (lkriffell) [inline spec/features/applications/show_spec.rb:58]: Nice job getting rid of unneeded code

File: app/views/pets/show.html.erb (hunk 1 above)
Comment (lkriffell) [inline app/views/pets/show.html.erb:3]: Awesome work

File: spec/features/applications/apply_spec.rb [hunk 3]
```diff
@@ -83,12 +83,6 @@
...
 
-    fill_in :name, with: "John"
-    fill_in :address, with: 21345
-    fill_in :city, with: "Denver"
-    fill_in :state, with: "CO"
-    fill_in :zip, with: 80025
-    fill_in :phone_number, with: "234-735-4743"
     fill_in :description, with: ""
 
     click_on 'Submit Application'
```
Comment (sarynm12) [inline spec/features/applications/apply_spec.rb:88]: Good work

**Review by sarynm12 — APPROVED** [review]

**Pull request merged**